src/Directory.Build.props

```diff
@@ -9,4 +9,64 @@
     <ShouldUnsetParentConfigurationAndPlatform>false</ShouldUnsetParentConfigurationAndPlatform>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <TargetOS>linux</TargetOS>
+    <TargetOS Condition="$([MSBuild]::IsOSPlatform('OSX'))">osx</TargetOS>
+    <TargetOS Condition="$([MSBuild]::IsOSPlatform('WINDOWS'))">windows</TargetOS>
+
+    <TargetArchitecture>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant)</TargetArchitecture>
+  </PropertyGroup>
+
+  <!-- MIBC data -->
+  <ItemGroup>
+
+    <!-- MIBC data to use when exact architecture match is available -->
+    <MIBCPackageDef 
```

Comment:
Just fyi, these are coming from arcade already.

Reply:
So AFAIU no, they are not coming from arcade, these don't seem to be set currently. Also, I borrowed this from the runtime, where they'd probably rely on arcade if it was the case.